### ISSUE TITLE
Suggestion – unicorn colourful icon options 

### ISSUE THREAD

**Issue by OrionRandD** [karlicoss/grasp#56]
I have used this extension for years...
But, it is difficult to see the unicorn (black and white) in my icon bar...
Could you add more unicorn colourful icons options for us to select and make a better contrast depending on the firefox/chromium icon bar background?

Thx a lot

**Comment by rosstimson** [comment]
I have this too.  When browser is set to match the OS in terms of light or dark mode dependent on the time of day the Grasp icon disappears when in dark mode.  Even a quick hack of giving it a white background would be better than virtually invisible.

**Comment by OrionRandD** [comment]
> I have this too. When browser is set to match the OS in terms of light or dark mode dependent on the time of day the Grasp icon disappears when in dark mode. Even a quick hack of giving it a white background would be better than virtually invisible.

Yes. In my case, I cannot use any firefox/chromium dark theme, because the icon just become invisible. But, it seems that our request have been ignored...  :(

**Comment by rosstimson** [comment]
@OrionRandD I've created a PR for this that is rudimentary but it at least works and makes things better. You might want to take a look. The screenshots in my PR are from Arc which is my primary browser but I assume it'll be similar in all browsers.

**Comment by OrionRandD** [comment]
> @OrionRandD I've created a PR for this that is rudimentary but it at least works and makes things better. You might want to take a look. The screenshots in my PR are from Arc which is my primary browser but I assume it'll be similar in all browsers.

Great! Thx a ton!
